cxx/fbjni/detail/CoreClasses-inl.h

```diff
@@ -364,13 +364,13 @@ inline ElementProxy<Target>& ElementProxy<Target>::operator=(const T& o) {
 }
```

Comment:
Need to fix T here as well, 2017 is probably only fine with it because it's not currently being used.